fre/yamltools/freyamltools.py

```diff
@@ -37,8 +36,12 @@ def function(context, uppercase):
                 type=str,
                 help="Target name",
                 required=True)
+@click.option("--use",
```

Comment:
I think you can put acceptable values here

Reply:
Yes, you can! Found how to by setting values with `type=click.Choice(['compile', 'pp'])`